tests/basic/tests.py

```diff
@@ -187,6 +189,19 @@ def test_save_parent_primary_with_default(self):
         with self.assertNumQueries(2):
             ChildPrimaryKeyWithDefault().save()
 
+    def test_save_deprecation(self):
+        a = Article(headline="original", pub_date=datetime(2014, 5, 16))
+        msg = "Passing positional arguments to save is deprecated"
+        with self.assertRaisesMessage(RemovedInDjango60Warning, msg):
+            a.save(False, False, None, None)
+            self.assertEqual(Article.objects.count(), 1)
+
+    @ignore_warnings(category=RemovedInDjango60Warning)
+    def test_save_positional_arguments(self):
+        a = Article(headline="original", pub_date=datetime(2014, 5, 16))
+        a.save(False, False, None, None)
+        self.assertEqual(Article.objects.count(), 1)
```

Comment:
Make sure the test you add fails without your changes.

Reply:
Sorry, I should have clarified that in the event your test case also passes on main (because we're not intending to change any substantive behavior here), then a test that increases coverage should be segregated into a separate commit.